src/operations/skeletons.js

```diff
@@ -180,7 +180,8 @@ export function makePreorderSkeleton(
   hashed.copy(opReturnBuffer, 3)
   opReturnBuffer.write(consensusHash, 23, 16, 'hex')
 
-  const nullOutput = bitcoin.script.nullData.output.encode(opReturnBuffer)
+  const nullOutput = bitcoin.payments.embed({ data: [opReturnBuffer] }).output
```

Comment:
again: `bitcoin.script` is gone. `bitcoin.payments` replaces it but with different calling conventions.